### ISSUE TITLE
[CKAN support] Can all of the non-native library files move into GameData?

### ISSUE THREAD

**Issue by JonnyOThan** [linuxgurugamer/SoundtrackEditorForked#7]
Someday soon I'd like to tackle the problem of installing native DLLs with CKAN (these typically go into KSP_x64_Data).  But SoundtrackEditorForked also installs a lot of files into the root KSP directory which is generally frowned on and doesn't seem to be strictly necessary.  Could you move these files into GameData somewhere?  If you don't want the automated game systems to try to load them then you can place them inside a PluginData directory in your mod folder; that will be skipped.

**Comment by linuxgurugamer** [comment]
I don't think that native DLLs will work inthe GameData.  They need to be in a specific location, AFAIK.  May be worth investigating,  but if it's about the only mod that does this (is it???), then I'm not sure it's worth putting the effort into CKAN to fix

**Comment by JonnyOThan** [comment]
Er right.  The native DLLs need to go in KSP_x64_Data (or the equivalent).  Some mods, I think principia, also put these in a PluginData folder and load them manually, but that's a bit trickier.

We will update CKAN to support this, because this isn't the only mod that needs it.

This issue is asking you to move everything ELSE (Music, Playlists, Licenses) somewhere inside the GameData directory because they don't need to be in the root KSP directory, right?  Once that's done and CKAN can support native DLLs, we'll be good to go.
